td.vue/src/router/index.js

```diff
@@ -40,7 +40,7 @@ const upgradeGuard = ((to, from, next) => {
         'Upgrade'
     ];
 
-    if (ignoreMatchers.some(x => to.name.indexOf(x) !== -1)) {
+    if (!to.name || ignoreMatchers.some(x => to.name.indexOf(x) !== -1)) {
```

Comment:
There was a bug where to.name was null and it threw a console error.  This could be handled better, but we should probably revisit routing errors before release (404s, permalinks)